src/components/common/ProductBox/ProductBox.js

```diff
@@ -77,7 +77,7 @@ const ProductBox = ({
           <Button variant='outline' onClick={compareHandler}>
             <FontAwesomeIcon
               icon={faExchangeAlt}
-              className={addCompare ? styles.addCompare : ''}
+              className={compareHandler ? styles.productIsCompared : ''}
```

Comment:
`compareHandler` to funkcja nie możesz jej tak użyć, to wyrażenie zawsze będzie zwracać `true`

Reply:
Druga sprawa to w zadaniu była informacja nt zmiany tła przycisku, ale do tego musisz przenieść ustawienie klasy na `<Button>` , bo w tym momencie uzyskany efekt nie jest zadawalający.

![image](https://user-images.githubusercontent.com/7238770/95810066-6a5c3e00-0d10-11eb-99f8-3c96f7cd33b2.png)